e2e/support/helpers/e2e-downloads-helpers.js

```diff
@@ -38,13 +40,13 @@ export function downloadAndAssert(
     getEndpoint(
       fileType,
       questionId,
-      publicUid,
+      publicUuid,
       dashcardId,
       dashboardId,
       isDashboard,
     );
-  const isPublicDownload = !!publicUid;
-  const method = isPublicDownload || isEmbed ? "GET" : "POST";
+  const isPublicDownload = !!publicUuid;
+  const method = (isPublicDownload && !dashcardId) || isEmbed ? "GET" : "POST";
```

Comment:
We use GET/POST a bit randomly for downloads, it's something I'll writ in a issue to standardize the download endpoints for team dashviz

Reply:
Hopefully it will eventually be all "POST" or all "GET" soon, is it worth changing it?

Reply:
well... that depends on how soon. If that change will come before the feature branch is merged to `master` then it's fine to keep it for the sake of convenience that we don't need to revert it back later on.

But if we don't exactly know when this change will come, then I'd just make the code easier to read right now.